x-pack/test/rule_registry/spaces_only/tests/trial/create_rule.ts

```diff
@@ -195,9 +196,23 @@ export default function registryRulesApiTest({ getService }: FtrProviderContext)
             ALERT_UUID,
             ALERT_RULE_EXECUTION_UUID,
             ALERT_RULE_UUID,
+            ALERT_TIME_RANGE,
             VERSION,
           ];
 
+          const alertUuid = alertEvent[ALERT_UUID]?.[0];
+          const executionUuid = alertEvent[ALERT_RULE_EXECUTION_UUID]?.[0];
+          expect(typeof alertUuid).to.be('string');
+          expect(typeof executionUuid).to.be('string');
+
+          await checkEventLogAlertUuids(
```

Comment:
this is where we check that the uuid in the AaD indices and the event log are the same